extensions/wikia/Email/Email.i18n.php

```diff
@@ -3,6 +3,7 @@
 
 $messages['en'] = array(
 	'emailext-desc' => 'An extension to handle delivering email',
+	'emailext-anonymous-editor' => 'A Wikia fan',
```

Comment:
This message was moved from WatchedPage.i18n.php since it could be applied to multiple email types.